pkg/registry/apis/query/register.go

```diff
@@ -115,6 +133,7 @@ func (b *QueryAPIBuilder) GetAPIGroupInfo(
 
 	storage := map[string]rest.Storage{}
 	storage[plugins.resourceInfo.StoragePath()] = plugins
+	storage["query"] = &subQueryREST{builder: b}
```

Comment:
does this work? i'd assume this would need to be something like:
```suggestion
	storage[plugins.resourceInfo.StoragePath()+"/query"] = &subQueryREST{builder: b}
```

Reply:
with "query" this gets attached to:
<img width="881" alt="image" src="https://github.com/grafana/grafana/assets/705951/9318a85d-dc9f-4419-85e2-206dabeee85a">

It is currently a Creator -- but I think we should change it back to a raw http handler because there is enough weird about it that I do not like (the response codes, logging, etc)

Reply:
ah ok, i was confused by the name `subQueryREST`